solutions/agents/variables.tf

```diff
@@ -1,12 +1,12 @@
 variable "ibmcloud_api_key" {
   type        = string
-  description = "The IBM Cloud api key."
+  description = "The IBM Cloud API key."
   sensitive   = true
 }
 
 variable "prefix" {
   type        = string
-  description = "(Optional) Prefix to append to all resources created by this solution."
+  description = "Optional. The prefix to append to the resources created by this solution."
```

Comment:
I've been pushing back in other repos about whether `Optional` is needed at all. It's very clear in the Cloud context, and i think it's just as clear to developers working directly in terraform.

Also, because developers know that `append` means to add something to the end, I don't use it for prefixes.

so, 

> The prefix for resources created by this solution.

or 
> The value to prepend to all resources created by this solution.

Reply:
I'd lean towards your first suggestion and have updated the text in the PR to reflect that.

"The prefix for resources created by this solution."